types/decimal.go

```diff
@@ -801,3 +801,8 @@ func MaxDec(d1, d2 Dec) Dec {
 func DecEq(t *testing.T, exp, got Dec) (*testing.T, bool, string, string, string) {
 	return t, exp.Equal(got), "expected:\t%v\ngot:\t\t%v", exp.String(), got.String()
 }
+
+func DecApproxEq(t *testing.T, d1 Dec, d2 Dec, tol Dec) (*testing.T, bool, string, string, string) {
```

Comment:
could you add a quick test or two and a changeling. ❤️

Reply:
Apologies, will do so ASAP!  Should I just add the tests in `decimal_test.go`?

Reply:
yes

Reply:
All required changes should have been made now.  Thanks so much!